packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -78,13 +78,18 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
   const [state] = useControl('NAVIGATE');
   const { path } = state;
 
-  const [{ data, isLoading, hasError }, handleList] = useAction('LIST_LOCATION_ITEMS');
+  const [{ data, isLoading, hasError }, handleList] = useAction(
+    'LIST_LOCATION_ITEMS'
+  );
   const [, handleLocationActionsState] = useControl('LOCATION_ACTIONS');
 
   const [, handleUpdateState] = useControl('LOCATION_ACTIONS');
 
+  const isFile = (file: File) => file.type !== undefined;
```

Comment:
can we put this outside the component so it doesn't get created on each render?

Reply:
Moved into the views util file